src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,63 @@
+package mate.academy.model;
+
+public class Book {
+    private long id;
+    private String title;
+    private int price;
+
+    private Book(long id, String title, int price) {
+        this.id = id;
+        this.title = title;
+        this.price = price;
+    }
+
+    private Book(String title, int price) {
+        this.title = title;
+        this.price = price;
+    }
```

Comment:
This constructor should be removed to maintain consistency as mentioned in the previous comment.